src/app/core/entity/entity-remove.service.spec.ts

```diff
@@ -169,7 +173,7 @@ describe("EntityRemoveService", () => {
     // @ts-ignore
     service.entityMapper = entityMapper;
 
-    await service.anonymize(entity);
+    await service[action](entity);
```

Comment:
I had left this in the tests when we removed in from the actual service because I can reuse the whole test setup/function easily. I'll see if I can remove this here as well when I refactor the tests as you suggested.